src/Bridge/Symfony/Bundle/Resources/config/api.xml

```diff
@@ -168,27 +168,27 @@
 
         <!-- Action -->
 
-        <service id="api_platform.action.placeholder" class="ApiPlatform\Core\Action\PlaceholderAction" />
-        <service id="api_platform.action.get_collection" alias="api_platform.action.placeholder" />
-        <service id="api_platform.action.post_collection" alias="api_platform.action.placeholder" />
-        <service id="api_platform.action.get_item" alias="api_platform.action.placeholder" />
-        <service id="api_platform.action.put_item" alias="api_platform.action.placeholder" />
-        <service id="api_platform.action.delete_item" alias="api_platform.action.placeholder" />
-        <service id="api_platform.action.get_subresource" alias="api_platform.action.placeholder" />
-
-        <service id="api_platform.action.entrypoint" class="ApiPlatform\Core\Action\EntrypointAction">
+        <service id="api_platform.action.placeholder" class="ApiPlatform\Core\Action\PlaceholderAction" public="true" />
+        <service id="api_platform.action.get_collection" alias="api_platform.action.placeholder" public="true" />
```

Comment:
Do we really need these aliases to be public? 🤔

Reply:
If they're used in routing, yes. At least they were public until now so I think they should still be when using sf 4.0.